README.md

```diff
@@ -1142,7 +1146,7 @@ Translations of the guide are available in the following languages:
   ```
 
 * <a name="no-multiline-ternary"></a>
-  Avoid multi-line `?:` (the ternary operator); use `if`/`unless` instead.
+  Line-split `?:` (the ternary operator) when it's clearer to do so.
```

Comment:
This seems like a less-authoritative stance, that makes it less clear when one should use `?:` I think it is better if it remained how it was.